app/presenters/outcome_presenter.rb

```diff
@@ -1,7 +1,25 @@
 require 'erubis'
 
 class OutcomePresenter < NodePresenter
-  class OutcomeTemplateMissing < StandardError; end
+  class ViewContext
+    def initialize(state)
+      @state = state
+      state_methods = state.methods - SmartAnswer::State.new(nil).methods
+      @state_reader_methods = state_methods.reject { |m| m =~ /=$/ }
+    end
+
+    def method_missing(method, *args, &block)
+      if @state_reader_methods.include?(method)
+        @state.send(method, *args, &block)
+      else
+        super
+      end
+    end
+
+    def get_binding
```

Comment:
That's cool. I was referring to this by the way https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/accessor_method_name.rb#L15-L19